File: CheckWeight.java
import java.util.Scanner;

// Check your weight program

//  @author: Oudone PKL

public class CheckWeight {

    // Method for check weight
    public static void checkWeight(int weight, int height) {
        weight = weight + 100 - height;
        System.out.println(weight);

        if(weight > 0) {
            System.out.println("You are obese.");
        }
    }
    
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter your weight: ");
        int weight = sc.nextInt();

        System.out.print("Enter your height: ");
        int height = sc.nextInt();

        // call the method
        checkWeight(weight, height);
    }
}

File: CheckWeight.py
# Check your weight program

# @author: Oudone PKL


# create a function 
def checkWeight(weight, height):
    n = 100
    weight = weight + 100 - height
    print(weight)
    if weight > 0:
        print("You are obese.")
    else:
        print("You are not obese.")

# input value from the user
weight = int(input("Enter your weight: "))
height = int(input("Enter your height: "))

# call function
checkWeight(weight, height)